File: CapstoneProject/AggregatorService/DataAccess/Models/UserData.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace AggregatorService.DataAccess.Models
{
    public class UserData
    {
        public int UserID { get; set; }
        public string Username { get; set; }
        public string Password { get; set; }
        public DateTime DateCreated { get; set; }
        public DateTime LastLogin { get; set; }
        public List<Topic> Topics { get; set; }

        public UserData()
        {
            Topics = new List<Topic>();
        }

        public UserData(UserData user)
        {
            UserID = user.UserID;
            Username = user.Username;
            Password = user.Password;
            DateCreated = user.DateCreated;
            LastLogin = user.LastLogin;
            Topics = user.Topics;
        }
    }
}

File: CapstoneProject/newsaggregator/src/Classes/UserData.js
export class UserData {
    UserID: number;
    Username: string;
    Password: string;
    DateCreated: Date;
    LastLogin: Date;

    constructor() {
        this.UserID = 0;
        this.Username = "";
        this.Password = "";
        this.DateCreated = null;
        this.LastLogin = null;
    }
}